en-US/Using_the_Fontforge_Drawing_Tools.md

```diff
@@ -101,7 +100,7 @@ paths:
 <p class="note"><b>Note:</b> As you can see, when setting breaks with a <i>Coin</i>, the
 direction of each handle should be tangent to the curve where it arrives.</p>
 
-## Mastering FontForge’s drawing tools
+## Mastering FontForge drawing tools
```

Comment:
Why?

Reply:
Probably there is no right or wrong as this is more of a stylistic thing.

Many companies have policies to always leave product names unchanged. (Some companies even go so far as to never use articles along with their product names.) Now, FontForge is not a commercial product but it is a product name nonetheless.